test/express-rabbitmq/tests/spec/app_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+RSpec.describe "Rabbitm app" do
```

Comment:
```suggestion
RSpec.describe "RabbitMQ app" do
```

(is this what it was supposed to say?)